dat/missions/zalek/neburesearch/neburesearch_03.lua

```diff
@@ -82,124 +94,218 @@ end
+    vn.na(_("They will surely come in handy. Is there anything else you want to do?"))
+    vn.func( function ()
+        player.pay(-glasses_price)
+        mem.has_glasses = true
+        vn.jump( "next_turn" )
+    end )
+
+    vn.label( "time_over" )
+    vn.na(_("It's time to go to the institute."))
+    vn.func( function ()
+        if mem.has_lab_coat then
+            mem.text1 = _("Wearing a lab coat, you blend in nicely with your audience.")
+            faction.modPlayerSingle("Za'lek", 1)
+        else
+            mem.text1 = _("It is just now that you realize that you're the only person not wearing a lab coat. This is a nightmare! Well, maybe it would be for a scientist?")
+        end
+        if mem.learned_text then
+            mem.text2 = _([[You start reciting the speech you got from the student. It turned out learning your text was the right choice. Some of the occurring terms are difficult to pronounce correctly.]])
+            faction.modPlayerSingle("Za'lek", 1)
+        elseif mem.has_glasses then
+            mem.text2 = _([[You start reciting the speech you got from the student. With the text being displayed on your glasses you don't have to look down on a paper to read the text, but some of the occurring terms are difficult to pronounce correctly.]])
+            faction.modPlayerSingle("Za'lek", 1)
+        else
+            mem.text2 = _([[You start reciting the speech you got from the student. You realize that you should have learned the text as some of the occurring terms are difficult to pronounce correctly.]])
+        end
+    end )
+    vn.na(_([[You make your way to the institute where you are supposed to give the talk, following the signs. On arriving you ask around where you should head to for your talk. Apparently you were already awaited. You are lead to a rather small seminar room. Finally you are ready to give your talk and a few scientist take seat. Actually you expected more than 17 listeners. Maybe this talk is not such a big deal as you expected. They wouldn't just send you to an important presentation, right?]]))
+    vn.na(mem.text1)
+    vn.na(mem.text2)
+    vn.na(_([[Finally, you are finished with your text and sigh in relief. After a brief applause, someone raises his hand. Apparently, he wants to ask a question. What should you do?
+You haven't understood the question, not even a single word.]]))
+    vn.menu( {
+        { _("This is a good question."), "avoid_question" },
+        { _("It's an open question."), "avoid_question" },
+        { _("Run!"), "run" },
+    } )
+
+    vn.label( "avoid_question" )
+    vn.na(_([[You try to avoid the question. The scientist seems to be upset. There are no further questions. Apparently, there is another talk that starts very soon so people hurry off. You manage to leave without gaining much attention and return to your ship. Enough science for today!]]))
+    vn.func( function ()
         faction.modPlayerSingle("Za'lek", 1)
```

Comment:
Would be probably better to move the standing modifications into the common script to make it easier to edit if necessary.